modules/electrophysiology_browser/jsx/react-series-data-viewer/src/series/components/EventManager.js

```diff
@@ -0,0 +1,145 @@
+// @flow
+
+import React from 'react';
+import type {Epoch as EpochType, RightPanel} from '../store/types';
+import {connect} from 'react-redux';
+import {setTimeSelection} from '../store/state/timeSelection';
+import {setRightPanel} from '../store/state/rightPanel';
+import * as R from 'ramda';
+import {toggleEpoch, updateActiveEpoch} from '../store/logic/filterEpochs';
+
+type Props = {
+  timeSelection: ?[number, number],
+  epochs: EpochType[],
+  filteredEpochs: number[],
+  setTimeSelection: [?number, ?number] => void,
+  setRightPanel: RightPanel => void,
+  toggleEpoch: number => void,
+  updateActiveEpoch: ?number => void,
+  interval: [number, number],
+};
+
+const EventManager = ({
+  epochs,
+  filteredEpochs,
+  setTimeSelection,
+  setRightPanel,
+  toggleEpoch,
+  updateActiveEpoch,
+  interval,
+}: Props) => {
+  return (
+    <>
```

Comment:
@laemtl I think the <> isn't needed since one div encompasses all the other elements.